homeassistant/components/notify/twitter.py

```diff
@@ -51,17 +54,96 @@ def __init__(self, consumer_key, consumer_secret, access_token_key,
                               access_token_secret)
 
     def send_message(self, message="", **kwargs):
-        """Tweet a message."""
+        """Tweet a message, optionally with media"""
+        media = kwargs.get('media', None)
+        media_id = self.upload_media(media)
```

Comment:
@balloob Can I do this in `[notify/__init__.py](https://github.com/MikeChristianson/home-assistant/blob/81a3be65398fe3732df04bf1b0b767ec29f77fb3/homeassistant/components/notify/__init__.py#L54)` so it applies to everything? Or must it be done, here and each other notifier?